src/js/utils/process-json.js

```diff
@@ -181,12 +187,19 @@ function processYoyData( data, group ) {
 /**
  * Returns a UTC timestamp number for the month when each graph's data is projected
  *
+ * For Mortgage Performance Trends data, there is no projected data.
+ * For Consumer Credit Trends data, projected data is for the last 6 months, except for inquiry index data, which is for the last 4 months, and inferred denials data, which is for the last _[@todo: add value once these charts are added]_ months.
+ *
  * @param {Array} valuesList - list of values from the data, containing an array with timestamp representing the month and year at index 0, and the value at index 1. Requires at least six months of data (six array items).
+ * @param {String} projectedRange - number of months in the data that is to be labeled projected. The default is 6 months, inquiry index charts are 4 months, etc.
  * @returns {Number} a timestamp.
  */
-function getProjectedTimestamp( valuesList ) {
-  // Projected data begins six months from the latest month of data available
-  const projectedMonth = valuesList[valuesList.length - 6][0];
+function getProjectedTimestamp( valuesList, projectedRange = 6 ) {
+  if ( projectedRange === 0 ) {
+    return false;
```

Comment:
Should have a consistent return type, so this should probably return `-1` (or `null` would also work I think).